fabric-tag-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/TagRegistry.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.entity.EntityType;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.item.Item;
+import net.minecraft.tag.GlobalTagAccessor;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.mixin.tag.BlockTagsAccessor;
+import net.fabricmc.fabric.mixin.tag.EntityTypeTagsAccessor;
+import net.fabricmc.fabric.mixin.tag.FluidTagsAccessor;
+import net.fabricmc.fabric.mixin.tag.ItemTagsAccessor;
+
+/**
+ * Helper methods for registering Tags.
+ */
+public final class TagRegistry {
+	private TagRegistry() {
+	}
+
+	public static Tag.Identified<Block> block(Identifier id) {
+		return getBlockAccessor().get(id.toString());
```

Comment:
Why not just use an Invoker to invoke vanilla's `register` methods directly?

Reply:
both have the same cost anyways. From experience with registries, we should not use the default register methods due to defaulting to minecraft namespace.